src/renderer/ui/components/generic/PlatformSpecific.js

```diff
@@ -1,14 +1,20 @@
 import { Component, PropTypes } from 'react';
+import os from 'os';
 
 export default class PlatformSpecific extends Component {
   static propTypes = {
     children: PropTypes.object,
     platform: PropTypes.string.isRequired,
+    version: PropTypes.string,
   };
 
   render() {
     if (process.platform === this.props.platform) {
-      return this.props.children;
+      if (!this.props.version) return this.props.children;
```

Comment:
If we're making a version check part of the components API it should probably be a semver check.  I'll fix it for now because I'm bored 😆 but in the future can we avoid scope creep on PR's.  I.e. A translation change shouldn't affect the component API.